latch/ldata/_transfer/remote_copy.py

```diff
@@ -10,24 +10,28 @@
 from .utils import query_with_retry
 
 
-def remote_copy(src: str, dst: str) -> None:
-    node_data = get_node_data(src, dst, allow_resolve_to_parent=True)
+def remote_copy(src: str, dst: str, create_parents: bool = False) -> None:
```

Comment:
This doesn't work yet, just opened PR in case anyone wants to do a quick initial pass

Reply:
currently fails with `ldata_node_no_slash_in_name`